packages/next/src/build/webpack/plugins/flight-manifest-plugin.ts

```diff
@@ -143,7 +147,7 @@ function entryNameToGroupName(entryName: string) {
     // Remove catch-all routes since they should be part of the parent group that the catch-all would apply to.
     // This is necessary to support parallel routes since multiple page components can be rendered on the same page.
     // In order to do that, we need to ensure that the manifests are merged together by putting them in the same group.
-    .replace(/\/\[?\[\.\.\.[^\]]*\]\]?/g, '')
+    .replace(/\/\[?\[\.\.\.[^\]]*]]?/g, '')
```

Comment:
This change seems unnecessary, is there a case where this doesn't match?

Reply:
my IDE was giving me a warning that this doesn't need to be escaped, I can undo though

(the regex has the same behavior)